public/app/features/playlist/ShareModal.tsx

```diff
@@ -16,9 +17,9 @@ export const ShareModal = ({ playlistUid, onDismiss }: Props) => {
   const [autoFit, setAutofit] = useState(false);
 
   const modes: Array<SelectableValue<PlaylistMode>> = [
-    { label: 'Normal', value: false },
-    { label: 'TV', value: 'tv' },
-    { label: 'Kiosk', value: true },
+    { label: t('share-modal.playlist.mode-normal', 'Normal'), value: false },
```

Comment:
`share-modal` refers to a different component. can we instead just create a new prefix to represent this different component, `share-playlist`? so the keys would be like `share-playlist.mode-normal`, `share-playlist.title`, `share-playlist.copy-link-button`, etc.

Reply:
Sure, I'll make the changes accordingly.